### ISSUE TITLE
Some bugs with Wagtail 2.0

### ISSUE THREAD

**Issue by SalahAdDin** [wagtail/wagtail#4222]
I have a group of recent bugs that i get with `wagtail` 2.0.

**First**: Explorer feature isn't working now.
![nocargapaginas](https://user-images.githubusercontent.com/5034215/35484017-4ddc78d8-045a-11e8-8853-2891cd90e9c2.gif)
Maybe it's because i had to build all static files since the beta version hasn't builded static files, but i'm not sure about it.
_Update_: This problem also is related to internet connection. I haven't this error in my University where internet connection, at least for me, is a monster XD.

**Second**: `Draftail` toolbar overlap other widgets.
![overlapping](https://user-images.githubusercontent.com/5034215/35484035-ca401fc4-045a-11e8-98be-e665e8157183.png)
I'm using a `DateField` before the `RichtextField` and as you can see `Draftail` toolbar is overlaping it.

**Third**: Side toolbar overlap Title page.
![otrosolapamiento](https://user-images.githubusercontent.com/5034215/35484070-7c72f838-045b-11e8-8422-b6b981aa6d35.png)
I think this happens because the sapnish translation is longer than english text.

**Fourth**: Migration created page haven't associated site.
When we create a page we have to associate it with a site, and after it create child page in order to get a correctly url routing(i think); we can see the next image if all is well.
![noproblemasaqui](https://user-images.githubusercontent.com/5034215/35484066-653ee9b0-045b-11e8-82f4-d4cc42bf2884.png)
Well, i create two page by migration (i know, it isn't advisable to do so), both are site associated page's childs (HomePage), but both of them haven't associated site, why?
![aquinohaysitio](https://user-images.githubusercontent.com/5034215/35484068-6a5a7216-045b-11e8-9c89-629f9ddd21c9.png)
If i create a child page from `wagtail` admin site i haven't any problem:
![acanohaylio](https://user-images.githubusercontent.com/5034215/35484192-c13d23ba-045d-11e8-9982-917f82207347.png)
I'm not sure about if this is a `wagtail 2.0` problem or not, because i didn't test it yet in older versions.
_Update:_ Yep, this is a bug with wagtail, i tested it with `1.13.1` and i'm still getting this bug.

**Fifth**: `Image` chooser pop up and upload `image` pop up aren't working correctly.
![deepin-screen-recorder_select area_20180128191847](https://user-images.githubusercontent.com/5034215/35484436-cb4fb522-0460-11e8-9b04-ae4b20db5f5b.gif)

Finally, I don't know if it's a problem with the beta or `django 2.0.1` but the site is very slow, very very slow. 
_Update_: I think this problem is related with my internet connection.

Thank you very much!

My current environtment:
* Python 3.6.4.
* Django 2.0.1.
* Wagtail 2.0b1.
* Chrome 64.

**Comment by thibaudcolas** [comment]
Thanks for taking the time to test this @SalahAdDin, that's super helpful ❤️.

I'll look into those issues later this week – for the exporter one I would be surprised if it was due to the changes in the new release because I've seen it work. For the datepicker z-index, I'll fix it. For the other ones – I'm clueless for now.

**Comment by gasman** [comment]
@SalahAdDin Thanks for the reports! Hopefully you've seen that the full beta package is released now (`pip install wagtail==2.0b1`) - please can you re-test with this version? I suspect this will fix issues 1 (explorer) and 5 (image chooser).

I can confirm the third bug (page title alignment) - it looks like this was introduced in 5f08013ad2d0a4c31f49fe4177eca4fad96539d5 . I'll open a new issue to cover this...

Regarding the fourth issue (pages created in a migration), I suspect your migrations are not setting the `path` and `depth` fields correctly. Please try running `./manage.py fixtree` - if this reports problems, your migrations need fixing.

**Comment by SalahAdDin** [comment]
@gasman Hi, i tried with these command and i get this:
```bash
/manage.py fixtree                                                                                                              1 ↵  225  15:44:01 
No problems found.
```

**Comment by SalahAdDin** [comment]
@gasman I can say that the internet connection's quality considerably affects the performance of the administrator: a poor internet connection affects all the functions that depend on `jQuery` or `React`; it is so that the explorer function does not work, the image choosers, document choosers, video choosers, etc. , and the autocomplete for tags (`TaggableClusters`) do not work correctly.

**Comment by thibaudcolas** [comment]
@SalahAdDin how poor of a connection are you talking about? All of those parts of the admin rely on AJAX requests, which shouldn't be any less reliable than other page loads (probably actually more reliable).

If you have specific problems here please take a moment to open a separate issue detailing them – generally we're moving to AJAX operations because they make for a **faster** experience, so if that's not the case we want to address it.

----

For the original issues, it seems everything is tackled apart from the datepicker + editor toolbar clash, which I'll add to #4229. I haven't been able to reproduce the explorer one but I imagine this was just due to the release not being available on pypi.

**Comment by gasman** [comment]
Fixed in #4229
